README.md

```diff
@@ -1 +1,188 @@
-# contact_form_final
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+    <h3><b>Portfolio </b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Portfolio <a name="about-project"></a>
+
+> This is the first html and css project of the first block in which linters are integrated.
+> This project is part of my Microverse exercises, where we learn how to set a up a new repository in a professional manners.
+> This project doesn't require any complex coding. The goal here is to master all of the tools and best practices that are used in a professional development environment.
+
+
+**Portfolio:** is a project where no complex coding is required and the only goal here is to master all of the tools and best practices we learned at Microverse.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/">index.html</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">style.html</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- User login form
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link (not yet deployed)](#)

```

Comment:
- [ ] Kindly remove the link and just add simple text without any link with the text "coming soon 🎈 ", right now your link is redirecting back to this PR which is not a good idea at all and makes your `documentation` looks unprofessional.